src/components/dialog/styles/dialog.css

```diff
@@ -0,0 +1,65 @@
+.main {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background: #FFF;
+    width: 450px;
+    min-height: 250px;
+    box-shadow: 0 0 7px 0px rgba(0, 0, 0, 0.25);
+    border-radius: 2px;
+    overflow: hidden;
+    font-family: sans-serif;
+    max-width: calc(100% - 30px);
+}
+
+.underlay {
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+}
+
+[data-underlay="true"] .underlay {
+    background: rgba(0, 0, 0, 0.3);
+}
+
+.title {
+    background-color: #34495e;
+    font-size: 20px;
+    color: #FFF;
+    padding: 14px 35px;
+    position: relative;
+}
+
+.content {
+    padding: 14px 35px;
+}
+
+.close {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    right: 35px;
+    cursor: pointer;
+    line-height: 1;
+}
+
+:global .dialogShow {
```

Comment:
@bitpshr Can we use `keyframe animations` for this please and move the `fadeIn` / `fadeOut` classes to an `animations.css` module perhaps alongside `variables.css` such that we can have consistent fadein / out across our widget suite. This approach also removes the need for the `:global` and `-active` class as `keyframes` provide a `from` and `to` for their animations. This is the approach used by the `animate.css` package and other popular animate libraries. The following works nicely:

``` css
@keyframes fadeIn {
  from {
    opacity: 0;
  }

  to {
    opacity: 1;
  }
}

.fadeIn {
   animation-duration: 0.2s;
  animation-fill-mode: both;
  animation-name: fadeIn;
}

@keyframes fadeOut {
  from {
    opacity: 1;
  }

  to {
    opacity: 0;
  }
}

.fadeOut {
  animation-duration: 0.2s;
  animation-fill-mode: both;
  animation-name: fadeOut;
}

```